gluten-core/src/main/scala/io/glutenproject/extension/ColumnarOverrides.scala

```diff
@@ -466,9 +465,15 @@ case class TransformPreOverrides(isAdaptiveContextOrTopParentExchange: Boolean)
         GenerateExecTransformer(plan.generator, plan.requiredChildOutput,
           plan.outer, plan.generatorOutput, child)
       case p =>
-        logDebug(s"Transformation for ${p.getClass} is currently not supported.")
-        val children = plan.children.map(replaceWithTransformerPlan)
-        p.withNewChildren(children)
+        val planTransformer = BackendsApiManager.getSparkPlanExecApiInstance
```

Comment:
why transforms HiveTableScanExec here ?

Reply:
Because I see `BatchScanExec`, `FileSourceScanExec` are all thansform in this `TransformPreOverrides` class,  `HiveTableScanExec` is similar to these,  so I make it here.  And it can not be used as `case HiveTableScanExec`, so I transform it like that.